assets/scss/templates/_integrations.scss

```diff
@@ -2,7 +2,7 @@
   margin-top: 20px;
 
   @at-root {
-    &__block {
+    /* &__block {
```

Comment:
What do you think about directly removing this code in place of commented it?

Reply:
Yes, at the moment I am doing tests to see it on the webapp, I promise to delete everything that is not used